tests/integration/components/sl-radio-group-test.js

```diff
@@ -0,0 +1,165 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-radio-group', 'Integration | Component | sl radio group', {
+    integration: true
+});
+
+test( 'The disabled state applies the disabled attribute and class', function( assert ) {
+    this.set( 'testName', 'test' );
+    this.render( hbs`
+        {{#sl-radio-group disabled=true name='testName'}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-radio-group' ).attr( 'disabled' ),
+        'disabled',
+        'has "disabled" attribute'
+    );
+
+    assert.ok(
+        this.$( '.sl-radio-group' ).hasClass( 'disabled' ),
+        'has "disabled" class'
+    );
+});
+
+test( 'The disabled state applies to sl-radio children', function( assert ) {
+    this.set( 'testName', 'test' );
```

Comment:
Assertion Failed: The name property must be set on the sl-radio-group component. its required